tests/system/src/test/java/test/javafx/accessibility/virtualflow/VirtualFlowMemoryLeakTest.java

```diff
@@ -0,0 +1,173 @@
+            stage.setScene(scene);
+            Platform.runLater(startupLatch::countDown);
+            stage.show();
+
+            Platform.accessibilityActiveProperty()
+                    .addListener((obs, prevActive, active) -> updateScreenReaderStatus(active));
+            updateScreenReaderStatus(Platform.isAccessibilityActive());
+        }
+    }
+
+    private static void updateScreenReaderStatus(final boolean active) {
+        if (active) {
+            screenReaderLatch.countDown();
+        }
+    }
+
+    private final static ObservableList<Item> items = FXCollections.observableArrayList();
+    private final static AtomicLong counter = new AtomicLong();
+
+    static class Item {
+
+        private final ReadOnlyObjectProperty<Long> id;
+
+        public Item(final long id) {
+            this.id = new SimpleObjectProperty<>(id);
+        }
+
+        public ReadOnlyObjectProperty<Long> idProperty() {
+            return id;
+        }
+    }
+
+    private static Parent createRoot() {
+        tableView = createTableView();
+        return tableView;
+    }
+
+    private static void addItem() {
+        items.add(0, new Item(counter.incrementAndGet()));
+
+        final TableViewSelectionModel<Item> sm = tableView.getSelectionModel();
+        if (sm.getSelectedItems().isEmpty()) {
+            sm.selectLast();
+        }
+        if (!tableView.isFocused()) {
+            tableView.requestFocus();
+        }
+    }
+
+    private static final AtomicBoolean itemsPopulatedAndCleared = new AtomicBoolean(false);
+    private static WeakReference<TableRow<Item>> firstRowRef;
+
+    private static TableView<Item> tableView;
+
+    private static TableView<Item> createTableView() {
+        final TableView<Item> tableView = new TableView<>(items);
+
+        tableView.setRowFactory(param -> {
+            final TableRow<Item> row = new TableRow<>();
+            if (itemsPopulatedAndCleared.get()) {
+                if (firstRowRef == null) {
+                    firstRowRef = new WeakReference<>(row);
+                }
+            }
+            return row;
+        });
+
+        final TableColumn<Item, Long> idColumn = new TableColumn<>();
+        idColumn.setCellValueFactory(cd -> cd.getValue().idProperty());
+
+        tableView.getColumns().add(idColumn);
+        return tableView;
+    }
+
+    @BeforeClass
+    public static void initFX() {
+        startupLatch = new CountDownLatch(1);
+        new Thread(() -> Application.launch(TestApp.class, (String[]) null)).start();
+        try {
+            if (!startupLatch.await(10, SECONDS)) {
+                fail("Timeout waiting for FX runtime to start");
+            }
+        } catch (final InterruptedException ex) {
+            fail("Unexpected exception: " + ex);
+        }
+    }
+
+    @AfterClass
+    public static void teardown() {
+        Platform.exit();
+    }
+
+    @Test
+    public void test_JDK8203345() throws Exception {
```

Comment:
Please also add the following, since this test is only applicable on Windows and Mac (no need to waste 5 seconds on a test that will not ever run on Linux).

```
    assumeTrue(PlatformUtil.isMac() || PlatformUtil.isWindows());
```

Reply:
Added copyright header and platform check. Furthermore removed the main method as requested by @premyi